docs/criteria/accelerators/beta.md

```diff
@@ -54,22 +52,22 @@ A minor bug is any bug that cannot be classified as a major bug.
   - List/FixedSizeList/LargeList
```

Comment:
Shall we just move "Core Arrow Data Types" to definitions section, and reference here and an in test coverage?